src/runtime_src/core/pcie/tools/xbutil/xbutil.cpp

```diff
@@ -1466,7 +1473,8 @@ int xcldev::device::runOneTest(std::string testName,
 	    std::cout << "INFO: == " << testName << " PASSED" << std::endl;
     } else if (ret == -EOPNOTSUPP) {
 	    std::cout << "INFO: == " << testName << " SKIPPED" << std::endl;
-        ret = 0;
+        if(testName.compare("verify kernel test") != 0)
```

Comment:
Sorry, could not figure out why this code was needed, especially since it is associated with a test name.